src/respend/respendrelayer.h

```diff
@@ -8,6 +8,8 @@
 
 namespace respend {
 
+static const int64_t DEFAULT_LIMITRESPENDRELAY = 100;
+
```

Comment:
seems like a lot of double spends allowed per minute...we only allow 30KB for free transactions, why so many for double spends?

Reply:
@dgenr8 ping!